src/config/webpack.base.ts

```diff
@@ -151,6 +154,7 @@ export default ({ mode, paths, env, sourceMaps }: IWebpackConfig): Configuration
                         },
                       ],
                       ['@babel/react', { development: !isProd, useBuiltIns: true }],
+                      ['@babel/typescript'],
```

Comment:
We've had some issues in the past over on API with the Typescript preset and class properties plugin when initializing static variables. This sort of explains it: https://github.com/babel/babel/issues/9105

We solved by adding the typescript plugin manually before class properties plugin: https://github.com/heydoctor/api/blob/master/.babelrc#L12

Reply:
https://github.com/babel/babel/issues/9105#issuecomment-555144941 🤔

Reply:
added below @babel/plugin-transform-typescript

Reply:
@vietnogigoodrx I think we'll want to remove the preset now that we're manually including the transform - preset looks to only include the transform plugin itself https://babeljs.io/docs/en/babel-preset-typescript

Reply:
Got it, removed